diff --git a/lib/index.js b/lib/index.js
index c5ca771c24dd914e342f791716a822431ee32b3a..4ae12aa2a6af0436f6c2dbb43ce00face6781479 100644
--- a/lib/index.js
+++ b/lib/index.js
@@ -132,7 +132,8 @@ function transformGfmAutolinkLiterals(tree) {
     tree,
     [
       [/(https?:\/\/|www(?=\.))([-.\w]+)([^ \t\r\n]*)/gi, findUrl],
-      [/(?<=^|\s|\p{P}|\p{S})([-.\w+]+)@([-\w]+(?:\.[-\w]+)+)/gu, findEmail]
+      // Replaced lookbehind (?<=...) with non-capturing group (2.0.0) for iOS Safari compatibility
+      [/([-.\w+]+)@([-\w]+(?:\.[-\w]+)+)/g, findEmail]
     ],
     {ignore: ['link', 'linkReference']}
   )
